facial-age-estimator/src/facial-age-estimator-model.js

```diff
@@ -0,0 +1,98 @@
+/* globals tf */
+import * as tf from '@tensorflow/tfjs-node'
+
+let modelPath = null
+
+if (!process.rollupBrowser) {
+  modelPath = `file://${__dirname}/../model/model.json`
+} else {
+  modelPath = 'https://max-assets.s3.us.cloud-object-storage.appdomain.cloud/facial-age-estimator/tf_js/0.1/model.json'
+}
+
+let model = null
+let warmed = false
+
+/**
+ * load the model
+ */
+const load = function (initialize) {
+  if (!model) {
+    console.log('loading model...')
```

Comment:
mind removing `console.log`? (or perhaps make it configureable with disabled/off as default)